build.sbt

```diff
@@ -53,6 +53,8 @@ lazy val baseSettings = Seq(
   //scalaVersion := "2.11.12",
   javacOptions ++= Seq("-source", "1.8", "-target", "1.8", "-encoding", "UTF-8", "-Xlint:-options"),
   javacOptions in doc := Seq("-source", "1.8"),
+  fork in Test := true,
```

Comment:
- https://xuwei-k.hatenablog.com/entry/2019/06/04/120316
- https://twitter.com/eed3si9n_ja/status/1135752128776069120
- https://github.com/sbt/sbt/pull/4769

workaround for sbt ClassLoader bug